src/Raven.Client/Documents/Operations/Counters/CounterBatch.cs

```diff
@@ -88,6 +88,7 @@ public class CounterOperation
         public long Delta;
 
         internal string ChangeVector;
+        public string DocumentId;
```

Comment:
Shouldn't this be internal?

Reply:
It fails on the TxReplay, since the serializing this ignore the internal.

Reply:
Can we fix the serialization, instead?
This is sitting on the client, and I don't want it exposed to users